src/mcp/tools/crashlytics/list_top_issues.ts

```diff
@@ -0,0 +1,39 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { listTopIssues } from "../../../crashlytics/listTopIssues.js";
+
+export const list_top_issues = tool(
+    {
+        name:"list_top_issues",
+        description: "List the top issues happening in the application.",
+        inputSchema: z.object({
+            platform: z.string(),
+            packageName: z.string(),
+            issueCount: z.string().optional(),
+        }),
+        annotations: {
+            title: "Get the list of top issues in the application",
+            readOnlyHint: true,
+        },
+        _meta: {
+            requiresAuth: true,
+            requiresProject: true,
+        },
+    },
+    async ({ platform, packageName, issueCount }, { projectId }) => {
+        if (projectId === undefined) {
+          return mcpError(`No projectId specified in the tool to get the list of issues.`);
```

Comment:
You don't need to check for this, it's handled by the surrounding harness.

Reply:
The challenge is because of the error where it is forcing us to check if ProjectID is defined before moving ahead. So, left this boiler plate check.